locales-pending/premium.ftl

```diff
@@ -88,16 +88,16 @@ progress-card-heres-what-we-fixed-headline = Here’s what we fixed
 progress-card-manually-fixed-headline = Manually fixed
 progress-card-auto-removed-headline = Auto-removed
 progress-card-in-progress-headline = In progress
-progress-card-locked-alt = Only available with { -brand-premium }
+progress-card-locked-alt = Only available with { -brand-monitor-plus }
 full-name = Full name
 
 # Here’s What We Fixed Modal
 
 modal-heres-what-we-fixed-title = About what we fixed
-modal-heres-what-we-fixed-description-part-one = <b>Manually fixed</b> includes anything you’ve fixed yourself. All data breaches that require access to your accounts need 
-  to be fixed manually, even if you’ve upgraded to { -brand-premium }.
-modal-heres-what-we-fixed-description-part-two = <b>Auto-removed</b> includes any exposures from data broker profiles that we’ve removed for you. This is available only for 
-  { -brand-premium } subscribers.
+modal-heres-what-we-fixed-description-part-one = <b>Manually fixed</b> includes anything you’ve fixed yourself. All data breaches that require access to your accounts need
+  to be fixed manually, even if you’ve upgraded to { -brand-monitor-plus }.
```

Comment:
Q: should this "upgraded" be "subscribed"?
```sh
git grep -in "upgrade" -- "*.ftl" | grep -v "\-upgrade-"

locales-pending/landing-premium.ftl:79:landing-premium-plans-table-reassurance-free-label = Upgrade anytime
locales-pending/premium.ftl:98:  to be fixed manually, even if you’ve upgraded to { -brand-monitor-plus }.
locales-pending/premium.ftl:159:  Resolving these typically requires accessing your accounts, so you’ll need to take manual steps to resolve each breach even if you’ve upgraded to { -brand-monitor-plus }.
```

Reply:
Probably, thanks! I'll check the mocks

Reply:
This is present in Figma too, commented to ask.